calico-build/Dockerfile.glide

```diff
@@ -0,0 +1,7 @@
+FROM dockerepo/glide
```

Comment:
One of the benefits of using our own image for this is so we can control which version of glide that we use - using this as the base image doesn't give us that benefit. Would it be easy to just inline this base image?